DigitalLearningSolutions.Web/ViewModels/Register/LearnerInformationViewModel.cs

```diff
@@ -10,12 +10,18 @@ public class LearnerInformationViewModel : IEditProfessionalRegistrationNumbers
     {
         public LearnerInformationViewModel() { }
 
-        public LearnerInformationViewModel(RegistrationData data)
+        public LearnerInformationViewModel(RegistrationData data, bool isSelfRegistration)

```

Comment:
Should this param be renamed to include the "OrEdit" of the property?

Reply:
I don't think so, this parameter only applies to a model being used for Registration journeys.